sd_hf_serve/app/run-model.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -x
+
+pip install --upgrade pip
+pip install diffusers==0.20.2 transformers==4.33.1 accelerate==0.22.0 safetensors==0.3.1 matplotlib Pillow ipython -U      
```

Comment:
Where is pip install for gradio?

Reply:
In [Dockerfile](https://github.com/yahavb/aws-neuron-eks-samples/blob/1a9740df7648d417af944e2760c84bb8ef071450/sd_hf_serve/app/Dockerfile.template#L12)